app/csv_generator/csv_generator.rb

```diff
@@ -8,19 +8,18 @@ class CsvGenerator
   # configuration => Pipelines::Configuration::Item
   attr_accessor :run, :configuration
 
-  # return a CSV file
+  # return a CSV String
   # using run and configuration attributes
   # to generate headers and data
   def generate_sample_sheet
     CSV.generate do |csv|
       csv << csv_headers
-      sorted_wells = sort_wells(run.plate.wells)
 
       sorted_wells.each do |well|
         # add well header row
         csv << csv_data(well: well, row_type: :well)
 
-        next unless well.all_libraries_tagged
```

Comment:
In practice I don't think we'll see significant changes here, as we *shouldn't* see a mix of tagged and untagged libraries, but this feels like a safer failure state.